doc/user_guide/mva.rst

```diff
@@ -104,11 +104,11 @@ the total variance specified, and the components above this value will be
 styled distinctly from the remaining components to show which are considered
 signal, as opposed to noise. Alternatively, by providing an integer value
 for ``threshold``, the line will be drawn at the specified component (see
-below).  The no of significant components can be estimated and a vertical line drawn to
-represent this by specifying "vline" as True. In this case the elbow or knee is found 
+below).  The number of significant components can be estimated and a vertical line drawn to
+represent this by specifying ''vline'' as True. In this case, the elbow or knee is found 
```

Comment:
Sorry, can you change this to use backticks (\`) instead of single quotes (like \`\`threshold\`\` above it)? That will cause the value to be formatted in monospace in the generated documentation.